core/src/main/scala/scheduler/KubernetesHttp.scala

```diff
@@ -0,0 +1,407 @@
+package nelson
+package scheduler
+
+import argonaut._
+import argonaut.Argonaut._
+import journal.Logger
+import org.http4s.{Method, Request}
+import org.http4s.Credentials.Token
+import org.http4s.AuthScheme
+import org.http4s.Status.NotFound
+import org.http4s.argonaut._
+import org.http4s.client.{Client, UnexpectedStatus}
+import org.http4s.headers.Authorization
+import scalaz.{~>, Foldable, Monoid}
+import scalaz.Scalaz._
+import scalaz.concurrent.Task
+
+import nelson.scheduler.SchedulerOp._
+import nelson.Datacenter.{Deployment, StackName}
+import nelson.Manifest.{EnvironmentVariable, Plan, Ports, Port, UnitDef, Versioned}
+import nelson.docker.Docker.Image
+
+object KubernetesHttp {
+  private val log = Logger[KubernetesHttp.type]
+}
+
+/**
+ * SchedulerOp interpreter that uses the Kubernetes API server.
+ *
+ * See: https://kubernetes.io/docs/api-reference/v1.8/
+ */
+final class KubernetesHttp(kubernetes: Infrastructure.Kubernetes, client: Client, serviceAccountToken: String) extends (SchedulerOp ~> Task) {
+  import KubernetesHttp.log
+  import KubernetesJson.{CronJobStatus, DeploymentStatus, JobReference, JobStatus}
+
+  def apply[A](fa: SchedulerOp[A]): Task[A] = fa match {
+    case Delete(dc, deployment) =>
+      delete(dc, deployment)
+    case Launch(image, dc, ns, unit, plan, hash) =>
+      launch(image, dc, ns, Versioned.unwrap(unit), unit.version, plan, hash)
+    case Summary(dc, ns, stackName) =>
+      summary(dc, ns, stackName)
+
+    // TODO: Legacy, no longer used
+    case RunningUnits(dc, prefix) =>
```

Comment:
This is actually used in `nelson stack runtime` I believe :-)

Reply:
Hm I tried tracing it in the codebase and also looking at https://verizon.github.io/nelson/reference.html#cli , I think that only uses `Summary` ? It seems `RunningUnits` is mostly used in `NomadHttp` so it can kill child processes for jobs and stuff ?

Reply:
My trace:

* [`/runtime` endpoint](https://github.com/Verizon/nelson/blob/master/http/src/main/scala/plans/Datacenters.scala#L234) calls `Nelson.getRuntimeSummary`
* [`Nelson.getRuntimeSummary`](https://github.com/Verizon/nelson/blob/master/core/src/main/scala/Nelson.scala#L592) calls `SchedulerOp.summary`